vic/vic_run/src/CalcBlowingSnow.c

```diff
@@ -363,24 +363,29 @@ polint(double  xa[],
  *****************************************************************************/
 double
 trapzd(double (*funcd)(),
-       double es,
-       double Wind,
-       double AirDens,
-       double ZO,
-       double EactAir,
-       double F,
-       double hsalt,
-       double phi_r,
-       double ushear,
-       double Zrh,
-       double a,
-       double b,
-       int    n)
+       double   es,
+       double   Wind,
+       double   AirDens,
+       double   ZO,
+       double   EactAir,
+       double   F,
+       double   hsalt,
+       double   phi_r,
+       double   ushear,
+       double   Zrh,
+       double   a,
+       double   b,
+       int      n)
 {
     double        x, tnm, sum, del;
     static double s;
+
     int           it, j;
 
+    // TODO: remove use of static variables (see GH #735), for now:
+    // make static variables thread safe
+    #pragma omp threadprivate(s)
```

Comment:
Do you have to put the pragma statement at the loop or can we place it near the declaration for `s`, so the link between `static` and the `threadprivate()` part is clearer? Same for other `#pragma omp threadprivate()`